### PR TITLE
Azure VM supportability

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3006

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/credential-guard/credential-guard-manage.md [hunk 1]
```diff
@@ -177,6 +177,9 @@ To disable Windows Defender Credential Guard, you can use the following set of p
 > [!NOTE]
 > The PC must have one-time access to a domain controller to decrypt content, such as files that were encrypted with EFS. If you want to turn off both Windows Defender Credential Guard and virtualization-based security, run the following bcdedit command after turning off all virtualization-based security Group Policy and registry settings: bcdedit /set {0cb3b571-2f2e-4343-a879-d86a476d7215} loadoptions DISABLE-LSA-ISO,DISABLE-VBS
 
+> [!NOTE]
+> Credential Guard and Device Guard are not currently supported when using Azure IaaS VMs. These options will be made available with future Gen 2 VMs.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/credential-guard/credential-guard-manage.md]: Copy edited, no changes.

**Comment by mypil** [comment]
@DaniHalfin - can you please assist to check, approve and merge the content for this PR based on issue #3006?

Thank you.

**Comment by mypil** [comment]
@DaniHalfin  - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #3006?

Thank you.

**Comment by DaniHalfin** [comment]
Looks good. Verifying with team before merge.

**Pull request merged**